README.md

```diff
@@ -12,7 +12,7 @@ https://hacks.mozilla.org/2019/04/teaching-machines-to-triage-firefox-bugs/
 
 - **backout** - The aim of this classifier is to detect patches that might be more likely to be backed-out (because of build or test failures). It could be used for test prioritization/scheduling purposes.
 
-- **bugtype** - The aim of this classifier is to classify bugs according to their type.
+- **bugtype** - The aim of this classifier is to classify bugs according to their type, with four labels: crash/memory/performance/security.
```

Comment:
Those are the labels we could gather automatically from a subset of bugs on Bugzilla. The plan is to add more types after manual labeling.

Reply:
thanks. clarified the explanation.